scripts/fuzz_shell.js

```diff
@@ -174,8 +174,12 @@ var imports = {
 
 imports = Asyncify.instrumentImports(imports);
 
+var module = new WebAssembly.Module(binary);
+
+console.log('module ok');
```

Comment:
Is this meant to be committed?

Reply:
Oh, no, this was from my investigations of that fuzz bug...